### ISSUE TITLE
Inlinescript

### ISSUE THREAD

**Issue by RafDelgadoUK** [flcdrg/PowerShellWixExtension#12]
How can you specify two script files to run on install and the other on uninstall as there seems no way to distinguish from individual script files in the install sequence?

**Comment by flcdrg** [comment]
You are correct. There isn't a way to do that at the moment. My initial thought is that we'd need to add a corresponding set of uninstall actions that mirror the existing ones, though maybe there's a tidier way to do it.

(for my own reference, see http://stackoverflow.com/a/17595951/25702 )

**Comment by ddurham2** [comment]
I think this problems comes down tot he fact that you can only specify where in the sequence that ALL scripts run and there you can only specify a single condition.  

This was a limitation for me also because I needed to run some scripts before installing files and some others after installing files but could not.. I had to manually run powershell instead of using this nice plugin.

**Comment by GZidar** [comment]
This PR https://github.com/flcdrg/PowerShellWixExtension/pull/25 (if accepted) should address this issue
